api/libproject/devstart.rb

```diff
@@ -2221,6 +2221,25 @@ def create_workbench_db()
   )
 end
 
+def create_or_update_workbench_db(cmd_name, args)
+  ensure_docker cmd_name, args
+  with_cloud_proxy_and_db_env(cmd_name, args) do |ctx|
+    create_workbench_db
```

Comment:
So this is just another entry point to what we aleready have, which makes sense.

One confusing thing is that this always runs a had-coded create_db.sql script. Which makes it feel like a wart in some ways. It's also scary to run something called `create_workbench_db` when you're just doing an update.

Reply:
Fair point on naming, renamed.

The choice for SQL and XML templates in our dev tooling is to use env substitution. For better or worse, that is the convention. I've added some comments to make the implicit parameterization that's happening here hopefully more clear.